uaclient/entitlements/repo.py

```diff
@@ -133,26 +134,34 @@ def can_disable(
 
         return result, reason
 
-    def _perform_enable(self, silent: bool = False) -> bool:
+    def enable_steps(self) -> int:
+        will_install = self.packages is not None and len(self.packages) > 0
+        if self.access_only or not will_install:
```

Comment:
Let's add a comment stating which operation we think should happen, just to make it clear why we have those numbers here